cmd/swarm/access_test.go

```diff
@@ -49,22 +50,52 @@ const (
 
 var DefaultCurve = crypto.S256()
 
-// TestAccessPassword tests for the correct creation of an ACT manifest protected by a password.
+const clusterSize = 3
+
+var clusteronce sync.Once
+var cluster *testCluster
+
+func initCluster(t *testing.T) {
```

Comment:
can we move this to `run_test.go`, where all the test helpers reside?

Reply:
done